### PR TITLE
Update Fork (#1)

### PR DESCRIPTION
**Pull request by CorentinGS**
* Blacklisted a server for a cmd

Blacklisted server BalanceTonQuoi ('583242472881127434') for the 'neko' glitch command.

* Added welcomer

Added welcomer for server Aperture (466600971213209600)

* Fixed indentation

* Fixed indentation

* Fixed indentation & selfed

* Update listener.py

Added BTQ welcomer

* Selfing

Selfed 'bot.get_guild'

* Update listener.py

Removed string & funcs join2

* Keywords update

Replaced 'name' with 'mention' in cuddle commands

* Edited welcomer

- removed BTQ welcomer
- edited welcome message for Aperture

* Update listener.py

Corrected end string misstyping

* Add a forum user command

Get information about aper.me forum's users.

* Fix indentation

* Fix content_type error

### CODE REVIEW

File: modules/fun.py [hunk 1]
```diff
@@ -88,11 +88,9 @@ def __init__(self, bot):
...
-        if 'neko' in message.content:
-            if message.guild.id != btq_id:
-                await message.channel.send(nekos.textcat())
-        if 'Neko' in message.content:
-            if message.guild.id != btq_id:
-                await message.channel.send(nekos.textcat())
+        if 'neko' in message.content and message.guild.id != btq_id:
+            await message.channel.send(nekos.textcat())
+        if 'Neko' in message.content and message.guild.id != btq_id:
+            await message.channel.send(nekos.textcat())
```
Comment (CorentinGS) [inline modules/fun.py:95]: Fusion des conditions

**Review by CorentinGS — COMMENTED** [review]
Amélioration des exceptions

**Review by TheApertureProject — APPROVED** [review]
Very well. I'm a bit new to the way you "beautify" stuff, but it seems acceptable to me. Let's see...

File: modules/fun.py (hunk 1 above)
Comment (TheApertureProject) [reply modules/fun.py:95]: Seems better this way.

**Comment by TheApertureProject** [comment]
I'll apply these changes right now and see if it works. 😄

**Comment by TheApertureProject** [comment]
Most changes were applied and are about to be tested soon. Thanks for the improvments Yume :)

**Pull request merged**